src/test/unit/test.js

```diff
@@ -59,6 +60,23 @@ test("Instantiate", function () {
               "Type of first WYMeditor instance, using jQuery.wymeditors(0)");
 });
 
+asyncTest("Custom dialog options don't change future defaults", function () {
```

Comment:
I read through it again and I'm not sure where comments would help. The name of the test succinctly explains what this is looking for and then the rest of the code has good variable names to explain what's being done/tested. Suggestions?